src/Concretize.hs

```diff
@@ -525,9 +526,9 @@ instantiateGenericStructType typeEnv originalStructTy@(StructTy _ _) genericStru
               nameFixedMembers = renameGenericTypeSymbolsOnProduct renamedOrig memberXObjs
               validMembers = replaceGenericTypeSymbolsOnMembers mapp nameFixedMembers
               concretelyTypedMembers = replaceGenericTypeSymbolsOnMembers mappings memberXObjs
-           -- We only used the renamed types for validation--passing the
-           -- renamed xobjs further down leads to syntactical issues.
-           in case validateMembers typeEnv renamedOrig validMembers of
+           in -- We only used the renamed types for validation--passing the
+              -- renamed xobjs further down leads to syntactical issues.
+              case validateMembers typeEnv renamedOrig validMembers of
```

Comment:
This is a weird reformat for me

Reply:
It's what Ormolu did...

Reply:
I can't find a good way around it, unfortunately 🤷‍♂️